docs/extensions/utility-functions.mdx

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 6
+sidebar_label: "Utilities"
+id: extension-utils
+title: "Utilities"
+description: Utilities for working with extensions
+slug: /extensions/utils
+---
+
+Kwil provides utilities for altering core network state from within extensions. With a database connection, you can use these utilities to get user balances,
```

Comment:
What is the package(s) that expose the utility functions? Not clear in these docs.